src/frontend/hooks/goose-form.js

```diff
@@ -81,13 +81,25 @@ const useGooseForm = () => {
 
   const saveForm = useCallback(async () => {
     // TODO: Validation checks here.
-
     // Send data to server:
+
     try {
+      // files added 
       const files = images;
-      let imageUrls = [];
-
-      if (files.length > 0) {
+  
+      // all files uploaded 
+      let imageUrls2 = [];
```

Comment:
What's the purpose of this array? Can we name it to something more descriptive, if needed?